### ISSUE TITLE
[M0-Dev] Problema con la estructura de las clases

### ISSUE THREAD

**Issue by manujurado1** [ignaciotitos/IV---HealthForAll#16]
Revisando la forma en la que se han organizado las clases, (según lo discutido en #14 y #13) he descubierto que no se han seguido buenas prácticas a la forma de estructurarlo. 
Ya que si la clase moneda es la entidad que manejará las otras clases, veo más lógico que la clase moneda tenga un atributo Recomendación y otro Predicción en vez de que estos tengan un atributo con el nombre de la moneda.
Dicho esto, y si lo ves bien, entiendo que una moneda va a tener un único objeto del tipo recomendación, ya que la recomendación es única, pero con los objetos de la clase predicción no sé si sería un único objeto por moneda o puede darse el caso de que sea una lista de ellos. Una vez resuelta esta duda, creo que la estructura quedará bien montada

**Comment by ignaciotitos** [comment]
Sí, así es. Predicción sólo va a haber una, y recomendaciones puede haber varias.
